mantidproject/documents
# Update manual testing issue links for Inelastic/Indirect/Direct

**Description**
This PR updates the links to the Indirect, Direct and Inelastic manual testing documentation inside of the template excel spreadsheet.

**To test**
Open the excel spreadsheet. Check the links are updates for the Indirect/Inelastic/Direct testing.

Run the following and check that a separate issue is created for `Indirect` and `Inelastic`:

```
conda activate manual-tests
python create_issues.py "Release 6.9" issue_template_ISIS.xlsx --dry-run
```
